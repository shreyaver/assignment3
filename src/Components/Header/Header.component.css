header {
    flex: 0.5;
    min-height: 100px;
    display: flex;
    flex-direction: row;
    justify-content: center;
    align-items: center;
}
.headerContent {
    flex: 1;
    padding: 0;
    width: 20%;
    display: flex;
    flex-direction: column;
    justify-content: space-evenly;
}
.circle {
    width: 75px;
    line-height: 75px;
    height: 75px;
    border: 2px solid white;
    border-radius: 50%;
    vertical-align: middle;
    text-align: center;
    font-size: 3rem;
    color: white;
}