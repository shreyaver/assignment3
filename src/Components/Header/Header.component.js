import React from 'react';
import './Header.component.css';

const Header = props => (
  <header>
    <div className="headerContent">
      <div className="circle">
    BS
      </div>
    </div>
  </header>
);
export default Header;
