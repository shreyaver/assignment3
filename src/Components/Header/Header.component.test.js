import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header.component';

describe('Header', () => {
  it('renders without crashing', () => {
    const header = renderer.create(<Header />).toJSON();
    expect(header).toMatchSnapshot();
  });
});
