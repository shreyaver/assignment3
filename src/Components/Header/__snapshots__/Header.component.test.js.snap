// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders without crashing 1`] = `
<header>
  <div
    className="headerContent"
  >
    <div
      className="circle"
    >
      BS
    </div>
  </div>
</header>
`;
